docs/toc.md

```diff
@@ -1394,7 +1394,8 @@
 ### [Using the Visual Basic Development Environment](visual-basic/developing-apps/using-ide/using-the-visual-basic-development-environment.md)
 #### [Visual Basic Settings](visual-basic/developing-apps/using-ide/settings.md)
 #### [Visual Basic IntelliSense Code Snippets](visual-basic/developing-apps/using-ide/intellisense-code-snippets.md)
-#### [Refactoring and Rename Dialog Box (visual-basic/developing-apps/Visual Basic)](visual-basic/developing-apps/using-ide/refactoring-and-rename-dialog-box.md)
+#### [Refactoring](https://docs.microsoft.com/en-us/visualstudio/vb-ide/refactoring-vb?toc=dotnet)
```

Comment:
is this preparing us to have the contextual toc? this doesn't work yet, right? also remove locale from these links on the toc file